config/install/core.entity_form_display.localgov_alert_banner.localgov_alert_banner.default.yml

```diff
@@ -0,0 +1,76 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.localgov_alert_banner.localgov_alert_banner.field_localgov_link
```

Comment:
Yes so if you compare this to https://github.com/localgovdrupal/localgov_services/blob/b8d65a1a361a05ec368f3156f7d818c12bc93fff/modules/localgov_services_page/config/install/core.entity_form_display.node.localgov_services_page.default.yml#L13

The field name `localgov_services_parent`, the bundle `localgov_services_page`, and the entity type `node`.